extensions/mssql/src/objectExplorerNodeProvider/hdfsProvider.ts

```diff
@@ -303,7 +304,7 @@ export class FileNode extends HdfsFileSourceNode implements IFileNode {
 			nodeStatus: undefined,
 			nodeType: Constants.MssqlClusterItems.File,
 			nodeSubType: this.getSubType(),
-			iconType: 'FileGroupFile'
+			iconType: this.isMounted() ? 'FileGroupFile_mounted' : 'FileGroupFile'
```

Comment:
>FileGroupFile_mounted [](start = 33, length = 21)

Wouldn't this be clearer as just File_mounted? Filegroup is a very different thing so might be confusion leaving it like this. Although I guess FileGroupFile is already like this so if you don't feel like updating both that's fine.